src/ubloxGPS.h

```diff
@@ -643,8 +649,8 @@ class ubloxGPS
     uint32_t getTime(void);
     uint32_t getUTCTime();
     uint8_t  getSatellites(void);
-    int8_t   getGeoIdHeight(void);
-    uint8_t  getHDOP(void);
+    double   getGeoIdHeight(void);
+    double   getHDOP(void);
```

Comment:
Would be good to at least add some description of the expected units and range.

Reply:
Meters. Added some doxygen comments